vsintegration/src/FSharp.Editor/LanguageService/LanguageService.fs

```diff
@@ -186,7 +181,7 @@ type internal FSharpWorkspaceServiceFactory [<Composition.ImportingConstructor>]
                                         enableBackgroundItemKeyStoreAndSemanticClassification,
                                     enablePartialTypeChecking = enablePartialTypeChecking,
                                     parallelReferenceResolution = enableParallelReferenceResolution,
-                                    captureIdentifiersWhenParsing = captureIdentifiersWhenParsing,
+                                    captureIdentifiersWhenParsing = enableFastFindReferences,
```

Comment:
This is set on startup, so in theory if you start with _fast find references_ off and enable it later in your session it won't have an effect until restart. Not sure if we care about that. 

We could also just set it to `true`. But I can imagine some hypothetical scenario where someone would want to disable it, maybe some generated files with so many unique identifiers in them that it would affect memory usage.

Reply:
In that case we should append " (requires restart) " to the `Enable_Fast_Find_References` option description in Strings.resx.

(wanted to make a suggestion, but GH does not allow me to suggest changing code outside of this diff)

Reply:
And possibly like this : ` (restart required, increases memory usage) `

Reply:
Yeah, added `restart required`. Not sure we want to scare people with `increases memory usage` which in almost all cases should be negligible.